src/System.Security.Permissions/ref/Configurations.props

```diff
@@ -8,6 +8,8 @@
     <BuildConfigurations>
       $(PackageConfigurations);
       netfx;
+      netcoreapp;
```

Comment:
This need to be in the PackageConfiguration statement. Both: netcoreapp and uap.